backend/organization/urls.py

```diff
@@ -144,4 +150,15 @@
         project_views.LeaveProject.as_view(),
         name='leave-project'
     )
+    ,
+    path(
+        'projects/<str:project_slug>/request_membership/<str:user_slug>/',
```

Comment:
This url is a bit weird to me. The User object is passed in an authenticated request anyways, so I don't see the need to pass the user_slug.

Reply:
Ok good to know. Will update and update call to the endpoint!

Reply:
I don't a problem with this URL. This specifically identifies that `user_id 1` has requested a member ship for `project_id 100`. More clearer. Its not about authentication but how the URL should be formed.

Reply:
We can fix this later since we have been working on this PR for a while.